### PR TITLE
Ignore errors while querying autoconfig urls.

### PR DESCRIPTION
**Pull request by Scheirle**
This removes most of the errors reported in #1513, but not all.

### CODE REVIEW

**Comment by ChristophWurst** [comment]
Wow. You're fast :o

**Comment by ChristophWurst** [comment]
Hm, I still see many errors. Can't tell if there are fewer when using this patch :-/

**Comment by Scheirle** [comment]
My mistake, wasn't in debug mode so the messages got suppressed.

I now pushed a version which removes the error handler and restores it afterwards.
I'm not sure if this is the best solution.

File: lib/service/autoconfig/ispdb.php [hunk 1]
```diff
@@ -48,15 +48,18 @@ public function __construct(Logger $logger) {
 
 	private function queryUrl($url) {
 		try {
+			set_error_handler(function() { /* ignore all errors */ });
```
Comment (ChristophWurst) [inline lib/service/autoconfig/ispdb.php]: I don't have a good feeling adding that :-/

**Comment by ChristophWurst** [comment]
I'm closing this PR due to inactivity
Thanks :-)

**Pull request closed without merging**